### ISSUE TITLE
Unable to BulkCopy Entities with a ConcurrencyToken

### ISSUE THREAD

**Issue by mpoqq** [linq2db/linq2db.EntityFrameworkCore#223]
We currently facing the issue that we are unable to BulkCopy Entities with a ConcurrencyToken.
The BulkCopy operation uses the ConcurrencyToken in the INSERT statement:
`INSERT INTO "BranchCoverageReadModelFiles"
(
        "xmin",
        "Name",
)
VALUES
(0,'test1'),
(0,'test2')
`
and following error is thrown:
```
 ---> Npgsql.PostgresException (0x80004005): 42703: column "xmin" of relation "TetTable" does not exist

POSITION: 49
   at Npgsql.Internal.NpgsqlConnector.<ReadMessage>g__ReadMessageLong|211_0(NpgsqlConnector connector, Boolean async, DataRowLoadingMode dataRowLoadingMode, Boolean readingNotifications, Boolean isReadingPrependedMessage)
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior, Boolean async, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteReader(CommandBehavior behavior, Boolean async, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteNonQuery(Boolean async, CancellationToken cancellationToken)
   at LinqToDB.Data.DataConnection.ExecuteNonQueryAsync(CancellationToken cancellationToken)
   at LinqToDB.Data.CommandInfo.ExecuteAsync(CancellationToken cancellationToken)
   at LinqToDB.DataProvider.MultipleRowsHelper.ExecuteAsync(CancellationToken cancellationToken)
   at LinqToDB.DataProvider.BasicBulkCopy.MultipleRowsCopyHelperAsync(MultipleRowsHelper helper, IEnumerable source, String from, Action`1 prepFunction, Action`3 addFunction, Action`1 finishFunction, CancellationToken cancellationToken, Int32 maxParameters, Int32 maxSqlLength)
   at LinqToDB.EntityFrameworkCore.LinqToDBForEFTools.BulkCopyAsync[T](DbContext context, IEnumerable`1 source, CancellationToken cancellationToken)
```
We tried to set `SkipOnInsert` on the xmin property (`[Column(SkipOnInsert = true)]`), but it looks like that it will be overridden by the Entity Mapping configuration (`modelBuilder.Entity<TestEntity>().UseXminAsConcurrencyToken();`).

**Comment by sdanyliv** [comment]
Could you provide Model for such entity?

**Comment by mpoqq** [comment]
I created a minimal reproducible example: https://github.com/mpoqq/npsql-broken-connections/tree/linq2db
And here is the model:
https://github.com/mpoqq/npsql-broken-connections/blob/644ae8abc1d324f192094222e8de90a605a3fc9e/BrokenConnection/CustomDbContext.cs#L19-L24

**Comment by sdanyliv** [comment]
Fixed in Release 6.7.1

**Comment by mpoqq** [comment]
Wow, thank you very much. That was fast 🚀 
It now works as expected.
